frontends/mit-open/src/pages/HomePage/HeroSearch.tsx

```diff
@@ -80,41 +177,67 @@ const HeroSearch: React.FC = () => {
     [navigate],
   )
   const isMobile = !useMuiBreakpointAtLeast("sm")
-  const placeholder = isMobile
-    ? "Search subject, dept, professor, or course"
-    : "Search for non-degree subject, dept, professor, or course"
   return (
-    <Stack direction="row">
-      <SearchContainer>
+    <HeroWrapper>
+      <TitleAndControls>
         <Typography
           typography={{ xs: "h3", md: "h1" }}
           sx={{ paddingBottom: 1 }}
         >
           Learn with MIT
         </Typography>
-        <Typography>A place for all non-degree learning.</Typography>
-        <SearchInputStyled
-          placeholder={placeholder}
-          size={isMobile ? "medium" : "hero"}
-          fullWidth
-          value={searchText}
-          onChange={onSearchChange}
-          onClear={onSearchClear}
-          onSubmit={onSearchSubmit}
-        />
-        <Typography variant="subtitle1" sx={{ marginBottom: 1 }}>
-          Popular Searches
+        <Typography>
+          Explore MIT's <Emphasized>Non-Degree Education</Emphasized>
         </Typography>
-        <Stack direction="row" columnGap={1} flexWrap="wrap" rowGap={1}>
-          {SEARCH_CHIPS.map(({ label, ...others }, index) => (
-            <ChipLink key={index} size="medium" label={label} {...others} />
-          ))}
-        </Stack>
-      </SearchContainer>
+        <ControlsContainer>
+          <SearchInput
+            placeholder="Search for courses, programs, learning, and teaching materials"
+            size={isMobile ? "medium" : "hero"}
+            fullWidth
+            value={searchText}
+            onChange={onSearchChange}
+            onClear={onSearchClear}
+            onSubmit={onSearchSubmit}
+          />
+          <LinksContainer>
+            <TrenderingContainer>
+              <Typography typography={{ xs: "subtitle4", md: "subtitle3" }}>
```

Comment:
We may want to add some margin to the right of the "Trending" text, as sometimes it runs up right against the edge of the search chip next to it.

<img width="223" alt="image" src="https://github.com/mitodl/mit-open/assets/12089658/7e9c0b99-5c36-45de-a655-f1b8dffc4efd">